cmd/ocitools/validate.go

```diff
@@ -337,6 +337,17 @@ func checkLinux(spec rspec.Spec, rootfs string, hostCheck bool) (msgs []string)
 			} else if spec.Linux.Namespaces[index].Type == rspec.MountNamespace {
 				mountExists = true
 			}
+		} else {
+			if hostCheck {
+				_, err := os.Stat(spec.Linux.Namespaces[index].Path)
+				if err != nil {
+					if os.IsNotExist(err) {
+						msgs = append(msgs, fmt.Sprintf("Path of %v not exist", spec.Linux.Namespaces[index].Type))
+					} else {
```

Comment:
On 08/17/2016 11:46 AM, W. Trevor King wrote:
> To avoid a layer of nesting:
> 
> |_, err := os.Stat(…) if os.IsNotExist(err) { … } else if err != nil { … } |

fixed, thanks.